### DISCUSSION TITLE
Can't generate call relationships correctly? Is there a problem with my use?

### DISCUSSION THREAD

**Discussion by Ador-able** [chanhx/crabviz#41] [Q&A]
Developing Unity, using the VSCode editor, searching for references can find the class that calls this function.

However, it is not possible to create association relationships for the graphs created for these class

![屏幕截图 2024-08-27 191836](https://github.com/user-attachments/assets/2e3a765f-fc4f-4a2f-8cda-f52a9ab72764)

![屏幕截图 2024-08-27 192012](https://github.com/user-attachments/assets/c5504228-b1f4-4e21-9d36-aa65fee340cc)

**Comment by chanhx** [comment]
Crabviz relies on the Call Hierarchy feature provided by language servers to draw edges, not by searching.

Are you using the C# extension? Seems it [doesn't support Call Hierarchy](https://github.com/dotnet/vscode-csharp/issues/6297). You can check it yourself: right click on a function, and select the "Show Call Hierarchy" option, see if there are any results.

**Comment by Ador-able** [comment]
ok It's a pity that such a good plugin cannot be used. Thank you for your reply
